### ISSUE TITLE
Error: Unexpected character '#'

### ISSUE THREAD

**Issue by kevlened** [developit/microbundle#42]
I'm unable to run the microbundle from master without getting the error above.

The last successful SHA is ff42a14921c23a96f7266834b20f1380c23ec764 (just before the upgrade to `sade@1.2.0`)

Env:
macOS 10.13.2
node 8.9.4
npm 5.6.0

Repro:
```
git clone https://github.com/developit/dlv.git
cd dlv
npm i
npm i --save-dev developit/microbundle
npm run build
```

**Comment by developit** [comment]
I believe @lukeed has fixed this in sade@1.3.1

**Comment by lukeed** [comment]
Yup, I think it'll be fixed in `sade@1.3.1`. if you could retry from master and/or download that version of Sade specifically, that'd be great!

**Comment by kevlened** [comment]
Excellent! This is resolved, thanks
